core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2432,7 +2477,13 @@ class KafkaApis(val requestChannel: RequestChannel,
           origin = AppendOrigin.COORDINATOR,
           entriesPerPartition = controlRecords,
           requestLocal = requestLocal,
-          responseCallback = maybeSendResponseCallback(producerId, marker.transactionResult))
+          responseCallback = errors => {
```

Comment:
Does an existing test cover this change to the error handling for the existing code?

Reply:
Yes. There are existing unit tests and integrations tests testing this path. Note that the tests that I've added also exercise this part.